keyserver/handler.go

```diff
@@ -56,3 +65,12 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 		TB:   tb,
 	}, nil
 }
+
+func (server *ConiksServer) StoreUserPoliciesToKV(up *p.ConiksUserPolicies) error {
```

Comment:
> However, in my latest implementation, I decided to not store all the data blob in the DB anymore since I see there is no reasons to do it. What do you think?

Oh, I missed that. Sorry! Let me go look.

> I would like to open separate issue for the key change function, though.

Absolutely, makes sense!

> Just a side-note, we should de-serialize the data blob to user's key, user policies before responding

If we do this, though, don't we lose a bit of privacy from the server? I.e. the server now has more information about what is stored in the blob. I'd like to loop @arlolra @Liamsi into this discussion, too.